ecosystem/typescript/sdk/README.md

```diff
@@ -5,12 +5,20 @@
 [![NPM Package Downloads][npm-image-downloads]][npm-url]
 
 ## Quickstart
+
 The public SDK downloaded from [npmjs](https://www.npmjs.com/package/aptos) is compatible with the [Aptos devnet](https://fullnode.devnet.aptoslabs.com). To start building, run below command in your project directory:
 
 ```bash
 yarn add aptos
 ```
 
+or use the browser bundle
+
+```
+// SDK is available through the global name `aptosSDK`
```

Comment:
I think it would be clearer if this said "window.aptosSDK" instead of "global name". Also it may be overlooked as a comment, consider moving it to a separate paragraph instead.